.github/workflows/plugin-packages.yml

```diff
@@ -169,6 +169,13 @@ jobs:
         use_auth_provider: true
         credentials: ${{ secrets.GKE_SA_KEY }}
 
+    - name: Run Preflight Unit Tests
```

Comment:
Update `Publish Preflight Tests Report` step in this job to upload JUnit report of this suite

Reply:
done